test_common/harness/kernelHelpers.cpp

```diff
@@ -1467,6 +1467,43 @@ int checkFor3DImageSupport( cl_device_id device )
     return 0;
 }
 
+int checkForReadWriteImageSupport(cl_device_id device)
+{
+    if (checkForImageSupport(device))
+    {
+        return CL_IMAGE_FORMAT_NOT_SUPPORTED;
+    }
+
+    auto device_cl_version = get_device_cl_version(device);
+    if (device_cl_version >= Version(3, 0))
+    {
+        // In OpenCL 3.0, Read-Write images are optional.
+        // Check if they are supported.
+        cl_uint are_rw_images_supported{};
+        test_error(
```

Comment:
Minor: I think this will work OK, but the usual pattern is to assign to a separate error code and then check it vs. embedding the call into the macro.  Because of the way the macro is written, I think this will call the function twice if the first call returns an error, which probably isn't intended.

Reply:
Update: no action required here.  PR #944 will fix the macro so the function only gets called once with this pattern.  Thanks!